test/e2e/data/infrastructure-docker/main/cluster-template-topology-runtimesdk-v1beta1/cluster-runtimesdk.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
This is a v1beta1 version of test/e2e/data/infrastructure-docker/main/cluster-template-upgrades-runtimesdk/cluster-runtimesdk.yaml

Reply:
Should we add this as a comment in the file at the top? (same for the other yamls we create in this PR)